CMake/External_OpenCV.cmake

```diff
@@ -200,6 +236,11 @@ if (CMAKE_COMPILER_IS_GNUCC AND NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 6)
   list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DENABLE_PRECOMPILED_HEADERS:BOOL=OFF)
 endif()
 
+# OpenCV 3.3.0 has an option to enable C++ 11
+if (fletch_BUILD_CXX11)
+  list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DENABLE_CXX11:BOOL=ON)
```

Comment:
Note only MSVC 2015 and 2017 fully support C++11
MSVC 2013 has partial support, but not all, will probably need to have a check somewhere if we still want to support 2013

here is a list of popular compilers and their initial version to implement all of C++11 :

GCC 4.8.1 and later
Clang 3.3 and later
MSVC 2015 and later